homeassistant/components/tahoma/__init__.py

```diff
@@ -48,6 +48,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     try:
         await client.login()
         devices = await client.get_devices()
+        places = await client.get_places()
```

Comment:
Not sure if the service will be ok with it but we could asyncio.gather here

Reply:
Since this is being worked on actively, if this suggestion makes sense, please do it in one of the other active PRs

Reply:
Thanks @bdraco, I wrote this down and will have a look at it for the future. Didn't think of this, so will need to have a look at the advantages.

Reply:
> Not sure if the service will be ok with it but we could asyncio.gather here 
What are the advantages off using gather?

Reply:
It will happen in parallel instead of series. Less time we spend waiting for integrations to setup/faster startup.

Reply:
@tetienne I will have a look if we won't face any API limits by doing those 'expensive' calls in parallel, otherwise I will add it in a future PR.

Reply:
It's just when we set-up the integration. It should be OK I think.